docs/changelog.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## Upcoming
+
+### Changed
+
```

Comment:
The changelog has been updated in another PR, there are some conflicts with that file that need to be resolved

Reply:
done.